.travis.yml

```diff
@@ -5,9 +5,13 @@ dist: xenial
 matrix:
   include:
     - python: 2.7
-      env: TOXENV=py27
+      env: TOXENV=py27-mlflow12
+    - python: 2.7
+      env: TOXENV=py27-mlflow17
+    - python: 3.6
+      env: TOXENV=py36-mlflow12
     - python: 3.6
-      env: TOXENV=py36
+      env: TOXENV=py36-mlflow17
```

Comment:
I'd be more confortable if (at least for one Python version) we actually tested all mlflow versions that are in the tox file. Tests are not taking that long, so maybe adding a test with `TOXENV=py36-mlflow{12,13,14,15,16,17}` might be easier than adding 4 new items to the travis matrix.

Reply:
Ah, I didn't realise this is an option, totally makes sense to use it!

Reply:
Hm, weird, locally that `TOXENV` works fine, but on Travis it was just running the last of the series, not all of them: For example: https://travis-ci.org/github/facultyai/mlflow-faculty/jobs/704694529

Reply:
For the time being I've done the semi-annoying but workable explicit listing of versions. That way indeed runs all the tests.

IMHO this might be better on a Github Actions-based test runner (feel like it is more "matrixable" in a way), but probably not worth moving at the moment.